app/views/devise/ichain_sessions/new.html.haml

```diff
@@ -19,6 +21,6 @@
       .form-group
         = button_tag(type: 'submit', class: "btn btn-success pull-right") do
           = content_tag(:strong, 'Sign In')
-  
+
     - if devise_mapping.ichain_registerable?
-      = link_to "Sign up", new_ichain_registration_path(resource_name)
+      = link_to "Sign up", 'https://idp-portal.suse.com'
```

Comment:
Could we instead change this line?

https://github.com/SUSE/hackweek/blob/f9db79bc042a5c2ac46fac8161430339693dbc20/config/initializers/devise.rb#L205

Or even eventually at a later point, remove the gem `devise_ichain_authenticatable` from the Gemfile?

https://github.com/SUSE/hackweek/blob/f9db79bc042a5c2ac46fac8161430339693dbc20/Gemfile#L45

Reply:
Huh?

Reply:
So we're still relying on iChain? I have no idea to be honest, this is why I asked if we could remove that gem.

Reply:
I think the gem is used to point to the login proxy, which despite the name is using the IDP login backend. `config.ichain_base_url` probably is used to tell it how to redirect back to the app.

Reply:
`ichain` is just bad wording. The setup is not ichain but still works like ichain.